doc/vxlan/EVPN/EVPN_VXLAN_MULTIHOMING.md

```diff
@@ -0,0 +1,1140 @@
+```
+
+No new extensions are required for FRR to display information about either the Ethernet-Segments or the remote MACs
+
+<a id="27-database-examples"></a>
+## 2.7 Database Examples
+
+<a id="271-config-db-examples"></a>
+### 2.7.1 CONFIG DB examples
+
+The following CLI configuration becomes the entries in CONFIG DB below:
+
+```
+config evpn_es add PortChannel01 00:11:22:33:44:55:66:77:88:99
+config evpn_es add PortChannel02 3 00:11:22:33:44:55
+```
+
+```
+"ETHERNET_SEGMENT_TABLE:PortChannel01": {
+  "interface"                 = "PortChannel01"
+  "esi_type"                  = 0
+  "type0_operator_config"     = 00:11:22:33:44:55:66:77:88:99
+  "type3_system_mac"          = ""
+  "type3_local_discriminator" = ""
+  "df-preference              = ""
+}
+"ETHERNET_SEGMENT_TABLE:PortChannel02": {
+  "interface"                 = "PortChannel02"
+  "esi_type"                  = 3
+  "type0_operator_config"     = ""
+  "type3_system_mac"          = 00:11:22:33:44:55
+  "type3_local_discriminator" = 3
+  "df-preference              = ""
+},
+```
+
+<a id="272-app-db-examples"></a>
+### 2.7.2 APP DB examples
+
+```
+"SPLIT_HORIZON_LIST_TABLE:PortChannel01": {
+  "vteps": "192.168.0.1,192.168.0.2",
+}
+```
+
+<a id="28-serviceability-and-debug"></a>
+## 2.8 Serviceability and Debug
+
+The existing logging mechanisms shall be used. The show commands and the redis table dumps can be used as debug aids.
+
+<a id="29-limitations"></a>
+## 2.9 Limitations
+
+- Split horizon list programming will not be supported on sonic-vs due to the lack of support of SHL by the Linux Kernel. That limitation may be removed when addressed properly.
+
+<a id="210-unit-test-cases"></a>
+## 2.10 Unit Test Cases
+
+- Configure ES, verify SHL is programmed, verify interface points to the SHL created
+- Add remote peer in ES, verify SHL is programmed
+- Remove remote peer from ES, verify updated SHL is programmed
+- Delete ES, verify interface no longer points to SHL and SHL is deleted
+
+<a id="3---support-for-df-election-role-programming"></a>
+# 3 - Support for DF Election Role Programming
+<a id="31-feature-requirements"></a>
+## 3.1 Feature Requirements
+
+The following requirements are addressed:
+
+1. Support designated forwarder programming
+2. Support BUM traffic blocking based on DF state
+
+<a id="311-configuration-and-management-requirements"></a>
+### 3.1.1 Configuration and Management Requirements
+
+There are no additional configuration and management requirements beyond those in section [2.1.1](#211-configuration-and-management-requirements).
+
+<a id="312-scalability-requirements"></a>
+### 3.1.2 Scalability Requirements
+
+There are no additional scalability requirements beyond those in section 2.1.2.
+
+<a id="313-sai-overview"></a>
+### 3.1.3 SAI Overview
+
+Add a new attribute "SAI_BRIDGE_PORT_ATTR_DF_BLOCK" in sai_bridge_port_attr_t to indicate that the bridge port is blocked to forward any BUM traffic received from the Tunnel. 
```

Comment:
As soon as I understand - we support only VLAN-VNI mapping and not the bridging between vxlan interface and some physical port. 
If this is true - we don't need this requirement, right?